### ISSUE TITLE
Fix failing CI checks for GNU Emacs 25

### ISSUE THREAD

**Issue by mtreca** [emacsorphanage/gnuplot#69]
CI checks for GNU Emacs 25 keep failing.

@conao3 is there anything we can do to fix these?

**Comment by yamashita-optarc** [comment]
Yes it is my fault but I cannot understand why.

**Comment by conao3** [comment]
Yes it is my fault but I cannot understand why.

**Comment by mtreca** [comment]
No worries, I will try to look into it today.

I replaced the Cask call with the native `make test` in my fork (see [here](https://github.com/mtreca/gnuplot/commit/fccfe3a59e0f1e9b3ce832279c5dc8c5026b907e)) and I have contrasting results:

- The ERT test failures are not picked up as failures, but the tests are now running properly (I had to fix the Makefile for this, see commit 5fd785c47570687661cd850e64b8f363d6c88b2d).
- I have issues in GNU Emacs 24 and 25 (`caddr` is not recognized for some reasons).
